load.go

```diff
@@ -264,6 +272,21 @@ func readFile(templateKey string, baseDir string, filePath string) string {
 
 }
 
+func readBinaryFile(templateKey string, baseDir string, filePath string) []byte {
```

Comment:
It feels like the function above this is doing the same thing, but simply returning to a string - I'm wondering if this warrants a completely new function or whether there can be an argument for parsing the body from a file with the return data type (and it defaults to a string).

Reply:
Ok, I will try to modify the existing readFile function.

Reply:
@jpatdalton Done, you can look at new changes.